app/controllers/spree/store_controller_decorator.rb

```diff
@@ -1,6 +1,9 @@
 class Spree::StoreController
   layout 'darkswarm'
 
+  include I18nHelper
+  before_filter :set_locale
```

Comment:
Would you mind adding test for this in `spec/controllers/spree/store_controller_spec.rb`?

Reply:
When I think that the it will more expensive to add the test than the value it provides I skip it.
My initial intuition about this test was right.
But well, if you ask. I just spent an hour trying to write this test without success.
Maybe you can help, this is my best try:

```
require 'spec_helper'

describe Spree::StoreController, type: :controller do

  include I18nHelper
  def params
    {locale: 'xp'}
  end

  def spree_current_user
  end

  controller(Spree::StoreController) do
    before_filter :unauthorized
    def index
      render text: ""
    end
  end

  it "redirects to home when unauthorized" do
    get :index
    response.should render_template("shared/unauthorized", layout: 'darkswarm')
  end

  it "calls set_locale before index" do
    expect(controller).to receive(set_locale)

    get :index
  end
end
```